Decompiler/Options.cs

```diff
@@ -42,6 +42,26 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option("tool", DefaultValue = false,
+            HelpText = "Enable generic vpk dump tool for all supported resources.")]
+        public bool VPKTool { get; set; }
+
+        [Option('x', "extract",
+            HelpText = "Extract only, do not decompile")]
+        public bool Extract { get; set; }
+
+        [Option('f', "filter",
+            HelpText = "Dump filter starting with path, ex: panorama\\")]
+        public string Filter { get; set; }
+
+        [Option('e', "filter_ext",
+            HelpText = "Dump filter extension(s), example: vcss_c,vjs_c,vxml_c")]
+        public string FilterExt { get; set; }       
+
+        [Option('s', "silent",
```

Comment:
I only added it for speed since even redirecting to null slows the process when there are 30k+ files. Will do.